lib/ids.js

```diff
@@ -1,7 +1,10 @@
 const _ = require('underscore'),
       logger = require('./logger');
 
-const DEFAULT_ID_LENGTH = 5,
+const ID_LENGTH_DOC_ID = 'shortcode-id-length',
```

Comment:
The only one I'm aware of is `sentinel-meta-data`, which is a bit unfortunate for what you're suggesting. FWIW I'd like to actually move `sentinel-meta-data`, `shortcode-id-length` as well as transition information (currently stored in docs) out into a separate sentinel DB. This would help that greatly I think (as it would also be an opportunity to rename `sentinel-meta-data` to something else).